eva-accession-import-automation/genome_downloader.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2020 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from get_assembly_report_url import *
+import os
+import pandas
+import sys
+import urllib.request
+import re
+import json
+from __init__ import *
```

Comment:
Unless there is really a lot of things to import I usually don't like the `import *`
```suggestion
from __init__ import logger
```